README.md

```diff
@@ -14,6 +14,21 @@ juju deploy local:trusty/k8s
 ```
 There are no configuration for the charm at this time.
 
+**Version**: Set the version of hyperkube image to deploy
+
+**cidr**: Set the IP range for the kubernetes cluster. eg: 10.1.0.0/16
+
+
+## State Events:
+
+ **Kubelet.available** - Cluster apiserver, and scheduler have been started
```

Comment:
I believe states are case sensitive as well should make this "kubelet.available"